src/main/java/com/cloudbees/jenkins/support/api/FileContent.java

```diff
@@ -79,7 +79,7 @@ public void writeTo(OutputStream os, ContentFilter filter) throws IOException {
     }
 
     @Override
-    public long getTime() throws IOException {
+    public long getTime() {
```

Comment:
The fix looks good, but is this a realistic scenario? I mean when do you expect an exception being thrown when retrieving the time?

Reply:
`FileContent#getTime` is never thrown. Which is why I removed it. [FilePathContent#getTime()](https://github.com/jenkinsci/support-core-plugin/blob/support-core-2.57/src/main/java/com/cloudbees/jenkins/support/api/FilePathContent.java#L87-L94) is what can throw an exception (reading the time through a remote channel).